test/lib/util/errata_test.rb

```diff
@@ -36,29 +36,60 @@ def setup
       @erratum2.save!
 
       @empty_erratum = Katello::Erratum.create(:errata_id => "empty", :pulp_id => "empty")
+
+      @modular_erratum = ::Katello::Erratum.find_by(errata_id: 'RHEA-2019-002')
+      @modular_rpm = ::Katello::Rpm.create(name: "foobarmod", filename: "foobarmod-1.3.4.rpm", pulp_id: 'foobarmodularrpm')
```

Comment:
nit: should this be defined in fixtures (like `@modular_erratum` was) or use this existing fixture? https://github.com/Katello/katello/blob/ea9570ae2e2529b54f20f113abdab86b83371040/test/fixtures/models/katello_rpms.yml#L40-L51

Reply:
Fixed!